pyopenms_viz/_core.py

```diff
@@ -430,6 +435,65 @@ def _kind(self):
         return "scatter"
 
 
+class SequencePlot(BasePlot, ABC):
+    """
+    Plot peptide sequence with matched fragments indicated.
+
+    Plot Specific Parameters
+    ------------------------
+    seq_col : string, optional
+        The name for sequence column
+    ion_annotation : string, optional
+        The name for the ion annotation column
+    color_annotation : string, optional
+        The name for the color annotation column
+    x_pos : float, optional
+        The center horizontal position of the peptide sequence.
+    y_pos : float, optional
+        The center vertical position of the peptide sequence.
+    spacing : float, optional
+        The horizontal spacing between amino acids.
+    seq_fontsize : str, optional
+        The font size of the amino acids.
+    frag_len : float, optional
+        The length of the fragment lines.
+    """
+
+    @property
+    def _kind(self):
+        return "sequence"
+
+    def __init__(
+        self,
+        data: DataFrame,
+        x,
+        y,
```

Comment:
How is `x` and `y` used for these type of plots? Are they still used?

Reply:
They are not used, just left them as placeholders because they are used in the other plots.